<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTableView {
}

QTableView::item {
    border-right: 1px solid #d6d9dc;
}

QTableView::item:selected {
    background-color: rgba(0, 102, 204, 100);
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QSplitter" name="verticalSplitter">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>16</number>
      </property>
      <widget class="ProjectTreeView" name="projectTreeView">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="branchBtn">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>147</width>
              <height>36</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <underline>true</underline>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    text-align: left;
    color: blue;
    background-color: transparent;
}
QPushButton:hover {
    color: red;
}</string>
            </property>
            <property name="text">
             <string>main</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="SearchLineEdit" name="searchEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>36</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="levelSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>true</bool>
          </property>
          <widget class="LogTableView" name="logTableView">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
          </widget>
          <widget class="MessageTextEdit" name="editMessage">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="FileTableView" name="fileTableView"/>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="labState">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="AllbranchCbox">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>所有分支</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RefreshBtn">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectTreeView</class>
   <extends>QTreeView</extends>
   <header>widget/ProjectTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>LogTableView</class>
   <extends>QTableWidget</extends>
   <header>widget/LogTableView.h</header>
  </customwidget>
  <customwidget>
   <class>FileTableView</class>
   <extends>QTableWidget</extends>
   <header>widget/FileTableView.h</header>
  </customwidget>
  <customwidget>
   <class>MessageTextEdit</class>
   <extends>QTextEdit</extends>
   <header>widget/MessageTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>SearchLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widget/SearchLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
